addons/l10n_cn/views/report_voucher.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="external_layout_boxed" inherit_id="web.external_layout_boxed" primary="True">
+            <xpath expr="//div[hasclass('o_boxed_header')]" position="replace">
+                <div class="o_boxed_header">
+                <div class="row mb8">
+                    <div class="col-3 mb4">
+                        <img t-if="company.logo" t-att-src="image_data_uri(company.logo)" style="max-height: 45px;" alt="Logo"/>
+                    </div>
+                </div>
+                </div>
+            </xpath>
+        </template>
+
+        <template id="report_voucher_document">
+            <t t-set="o" t-value="o.with_context(lang=lang)" />
+            <t t-if="not company">
+                <!-- Multicompany -->
```

Comment:
All this to find the company, seems really unconventional.  Where does it come from?  Normally you just take o.company_id (on the account_move (which is a related with the journal))